public/static/docs/tutorials/versioning.md

```diff
@@ -15,8 +15,8 @@ to build a powerful image classifier using a pretty small dataset.
 
 We first train a classifier model using 1000 labeled images, then we double the
 number of images (2000) and retrain our model. We capture both datasets and
-classifier results and show how to use `dvc checkout` along with `git checkout`
-to switch between different versions.
+classifier results and show how to use `dvc checkout` to switch between model
```

Comment:
`both datasets and models` ... so correction narrows it inaccurately

Reply:
This sentence refers to the section further down titled **Switching between model versions**. Sure, data is also checked out but the point is the models. Should I change here and rename that header?

Reply:
OK its "data or model" in both places now. See cbecbb36